material-darker.xml

```diff
@@ -0,0 +1,34 @@
+		<key name=".Vanilla" modified="2017-02-15 13:30:28" build="160724">
+			<value name="ColorTable00" type="dword" data="00151515"/>
+			<value name="ColorTable01" type="dword" data="00ffaa82"/>
+			<value name="ColorTable02" type="dword" data="00187626"/>
+			<value name="ColorTable03" type="dword" data="00d6ccb2"/>
+			<value name="ColorTable04" type="dword" data="000098ff"/>
+			<value name="ColorTable05" type="dword" data="00b380bb"/>
+			<value name="ColorTable06" type="dword" data="006779ab"/>
+			<value name="ColorTable07" type="dword" data="00ffffee"/>
+			<value name="ColorTable08" type="dword" data="00b8b0a1"/>
+			<value name="ColorTable09" type="dword" data="00ffaa82"/>
+			<value name="ColorTable10" type="dword" data="008de8c3"/>
+			<value name="ColorTable11" type="dword" data="00ffdd89"/>
+			<value name="ColorTable12" type="dword" data="007053ff"/>
+			<value name="ColorTable13" type="dword" data="007871f0"/>
+			<value name="ColorTable14" type="dword" data="006bcbff"/>
+			<value name="ColorTable15" type="dword" data="00f5f5f5"/>
+			<value name="ColorTable16" type="dword" data="00000000"/>
+			<value name="ColorTable17" type="dword" data="00800000"/>
+			<value name="ColorTable18" type="dword" data="00008000"/>
+			<value name="ColorTable19" type="dword" data="00808000"/>
+			<value name="ColorTable20" type="dword" data="00000080"/>
+			<value name="ColorTable21" type="dword" data="00800080"/>
+			<value name="ColorTable22" type="dword" data="00008080"/>
+			<value name="ColorTable23" type="dword" data="00c0c0c0"/>
+			<value name="ColorTable24" type="dword" data="00808080"/>
+			<value name="ColorTable25" type="dword" data="00ff0000"/>
+			<value name="ColorTable26" type="dword" data="0000ff00"/>
+			<value name="ColorTable27" type="dword" data="00ffff00"/>
+			<value name="ColorTable28" type="dword" data="000000ff"/>
+			<value name="ColorTable29" type="dword" data="00ff00ff"/>
+			<value name="ColorTable30" type="dword" data="0000ffff"/>
+			<value name="ColorTable31" type="dword" data="00ffffff"/>
```

Comment:
It looks like the theme is missing several common elements.

Particularly Name, ExtendedColors, ExtendColorIdx, TextColorIdx, BackColorIdx, PopTextColorIdx, PopBackColorIdx.

You might compare this to another one of the themes to verify - moe-dark